### ISSUE TITLE
Until rclone wont be updating new files/dirs in mount, there is an bash script re-mount idea :)

### ISSUE THREAD

**Issue by scriptzteam** [ncw/rclone#790]
Call it blahblah.sh you can also add it into cron when fixed, btw you need to edit it by your paths etc ;)

```
pkill -f "mount"
fusermount -u /_mnt/
umount -f /_mnt/
umount -l /_mnt/
cd ~/rclone/rclone-v1.33-64-gbc414b6β-linux-386 && ./rclone mount --no-modtime secret:/FTP/ /home/remoteacd/ &
```

Not works as i expected but you can get idea, anyone can update to get it working, i am too lazy r8 now xD thx! If no-one i will post update/fix soon.

**Comment by scriptzteam** [comment]
Okay so fixed version - https://github.com/scriptzteam/rclone-re-mount/tree/master

I´ve added it into crontabs and it is run every 5 min. so i get everytime fresh content from ACD in my mount folder :)

**Comment by ncw** [comment]
Most ingenious!

I recommend you don't kill `-9` `rclone mount` as it has cleaning up it should do.  Try `fuser -k -15 /path_to/mount/` would be better.

(I'm going to fix this properly in #680 hopefully!)

**Comment by scriptzteam** [comment]
I needed to kill it with -9 since sometimes it "hang out" with "Device is busy blahblah" :) But thats just me xD

I am running this aka "remounting" via cron for 3 days and it works veeeery well :D

**Comment by ncw** [comment]
I have fixed #680 so I'm going to close this too.
